ts/Extensions/RangeSelector.ts

```diff
@@ -1459,7 +1459,7 @@ class RangeSelector {
             );
         }
 
-        if (time && useUTC) {
+        if (time && useUTC && isNumber(date)) {
```

Comment:
Could you tell us (or leave a comment?) why `date` can be NaN?

Reply:
Date.parse can return NaN

Reply:
Ok, then what does this change do? If `date` can be NaN, then no matter what we do here, we will return `NaN` anyway. Maybe I'm missing something?

Reply:
It prevents calling `getTimezoneOffset` with a NaN timestamp, which may be a user-provided callback, which is what the issue was all about. We dont really need the offset if we have no date to apply it to anyway. Returning NaN is fine.

Reply:
Thanks for the explanation! :) 

I will leave the decision to Torstein, but I think this might be unnecessary and may cause some issues in the future. If any similar issue (like the one above, with empty data series and dataGrouping) shows up, in the bug report we will have the title: "`getTimezoneOffset` not called" instead of "`getTimezoneOffset` called with NaN's" - not a bug difference. This only hides the real culprit, looks like a workaround 🤔